### DISCUSSION TITLE
Another reason for mismatch between number of documents found

### DISCUSSION THREAD

**Discussion by carry2web** [microsoft-search/pnp-modern-search#3735] [General]
For this page: docs/using-query-tools-to-verify-issues.md I would like to propose to add the libraries that are only showing major versions to visitors and not draft documents

**Comment by kasperbolarsen** [comment]
Are you suggesting to add that as yet another reason alongside these?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/5fbc17e8-6549-44a2-a5d4-c97dd13552d6)

**Comment by carry2web** [comment]
Yes

Op wo 8 mei 2024 16:19 schreef Kasper Larsen ***@***.***>:
